tutorials/workflow/csharp/task-chaining/TaskChaining/Program.cs

```diff
@@ -0,0 +1,25 @@
+using Dapr.Workflow;
+using TaskChaining;
+using TaskChaining.Activities;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddDaprWorkflow(options => {
+    options.RegisterWorkflow<ChainingWorkflow>();
+    options.RegisterActivity<Activity1>();
+    options.RegisterActivity<Activity2>();
+    options.RegisterActivity<Activity3>();
+});
+var app = builder.Build();
+
+app.MapPost("/start", async () => {
+    await using var scope  = app.Services.CreateAsyncScope();
+    var workflowClient = scope.ServiceProvider.GetRequiredService<DaprWorkflowClient>();
```

Comment:
Nope, as in a previous message- this isn't necessary. Inject `DaprWorkflowClient` into the async method (can use an optional `[FromServices]` attribute if you want) and access the instance from there. A separate scope is created by invocation of the inbound post request itself and isn't needed here.